tsconfig_templates/README.md

```diff
@@ -0,0 +1,5 @@
+# `tsconfig` Templates
```

Comment:
nit: let's use `tsconfig-templates` as the folder name to better match JS world

Reply:
Done.